### PR TITLE
feat: Add support for anonymous attribute redaction in feature events

### PR DESCRIPTION
**Pull request by keelerm84**

### CODE REVIEW

File: mockld/roku.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func (srv *RokuServer) Wrap(h http.Handler) http.Handler {
 			return
 		}
 
-		userJSONBase64 := base64.StdEncoding.EncodeToString(userJSON)
+		userJSONBase64 := base64.URLEncoding.EncodeToString(userJSON)
```
Comment (cwaldren-ld) [inline mockld/roku.go:31]: is this an unrelated bugfix?

File: sdktests/client_side_events_eval.go [hunk 2]
```diff
@@ -127,6 +129,133 @@ func doClientSideFeatureEventTests(t *ldtest.T) {
...
+			t.DebugLogger().Println(2)
+
+			for _, valueType := range getValueTypesToTest(t) {
+				t.Run(testDescFromType(valueType), func(t *ldtest.T) {
+					t.DebugLogger().Println(3)
+					userContext := userContextFactory.NextUniqueContext()
+					orgContext := orgContextFactory.NextUniqueContext()
+
+					multiContext := ldcontext.NewMultiBuilder().Add(userContext).Add(orgContext).Build()
+					t.DebugLogger().Println(4)
```
Comment (cwaldren-ld) [inline sdktests/client_side_events_eval.go:206]: Should the debug logs remain?

File: mockld/roku.go (hunk 1 above)
Comment (keelerm84) [reply mockld/roku.go]: That other PR landed the fix on v2 so we can get the benefit immediately. I had to wait for it to land, pull those changes into the feat/inline-contexts branch, and then update this PR accordingly.

tl;dr -- removed!

File: sdktests/client_side_events_eval.go (hunk 2 above)
Comment (keelerm84) [reply sdktests/client_side_events_eval.go]: Nope and removed.

**Review by cwaldren-ld — APPROVED** [review]
